docs/source/algorithms/LoadILLDiffraction-v1.rst

```diff
@@ -119,6 +119,8 @@ Output:
 
   The output has 1 bin (detector scan) and 1281 spectra
 
+<<<<<<< HEAD
```

Comment:
I think these are trace of previous merge conflict. Can they be fixed ?

Reply:
Absolutely, will fix it immediately. At the same time, there's an obsolete IN5 example that somehow got stuck there, so I am going to remove them both simultaneously.